change-notes/1.19/analysis-cpp.md

```diff
@@ -6,14 +6,16 @@
 
 | **Query**                   | **Tags**  | **Purpose**                                                        |
 |-----------------------------|-----------|--------------------------------------------------------------------|
-| *@name of query (Query ID)* | *Tags*    |*Aim of the new query and whether it is enabled by default or not*  |
+| Cast from char* to wchar_t* | security, external/cwe/cwe-704, external/microsoft/c/c6276 | Detects potentially dangerous casts from char* to wchar_t*.  Enabled by default on LGTM. |
```

Comment:
Do you think we want those "external/microsoft" tags in this file? I left them out of #235 because I thought they were of too narrow interest and too numerous (one query had 5 of them).

Reply:
Yeah, I don't think it's worth mentioning here.

Reply:
Done.